lib/rbs/cli.rb

```diff
@@ -750,5 +751,50 @@ def parse_type_name(string)
         TypeName.new(name: last, namespace: namespace.parent)
       end
     end
+
+    def test_opt options
+      opt_string = options.dirs.map { |dir| Shellwords.escape("-I #{dir}")}.concat(options.libs.map { |lib| Shellwords.escape("-r{lib}")}).join(' ')
+      opt_string.empty? ? nil : Shellwords.escape(opt_string)
```

Comment:
I don't think we need `Shellwords.escape(opt_string)` again here. Do we need it?